src/nncf/quantization/algorithms/smooth_quant/algorithm.py

```diff
@@ -98,6 +102,10 @@ def _set_backend_entity(self, model: TModel) -> None:
             msg = f"Cannot return backend-specific entity because {model_backend.value} is not supported!"
             raise nncf.UnsupportedBackendError(msg)
 
+        # Only the OpenVINO backend supports in-place statistics, so we should set this variable here.
+        if model_backend != BackendType.OPENVINO:
+            self._inplace_statistics = False
```

Comment:
`self._inplace_statistics` is originally set in the constructor and overwriting this variable like that may results in unexpected behavior. Perhaps, we should raise an exception if given value is not supported by the backend?

Reply:
Also, I thought only ONNX backend is affected, but with such condition it seems that all backends other than OpenVINO do not support in-place statistics. Is that right?